apps/src/blockly/addons/cdoFieldFlyout.ts

```diff
@@ -1,14 +1,18 @@
-import GoogleBlockly, {Block, FieldConfig, WorkspaceSvg} from 'blockly/core';
+import * as GoogleBlockly from 'blockly/core';
 
 import CdoBlockFlyout from './cdoBlockFlyout';
 
+interface FieldConfig {
+  tooltip?: string;
```

Comment:
Same question here.

Reply:
Can we reference https://github.com/google/blockly/blob/60da7d87625080ee44093c45a56689456883d6f6/core/field.ts#L1469?